src/Admin/Models/ProviderOrganizationItemViewModel.cs

```diff
@@ -0,0 +1,8 @@
+﻿using Bit.Core.Models.Data;
+
+namespace Bit.Admin.Models;
+
+public class ProviderOrganizationItemViewModel : ProviderOrganizationUnassignedOrganizationDetails
```

Comment:
Suggestion only: you could make this generic, e.g. `SelectableViewModel<T> : T`. It would make this a bit more useful generally and also explain what it is (i.e. a wrapper that just adds the `Selected` property).

Reply:
`SelectableViewModel<T> : T` sadly this doesn't work. I couldn't figure out a generic way to make this work